data/STRchive-loci.json

```diff
@@ -224,9 +224,9 @@
   "pathogenic_max": 68.0,
   "ref_copies": 34.0,
   "motif_len": 3,
-  "age_onset": "Typical: 20-49 [@pmid:11436124], Range: 8 [@pmid:15851746] - 78 [@pmid:19227892].",
+  "age_onset": "Typical: 20-49 [@pmid:11436124], Range: 8 [@pmid:15851746] - 83 [@doi:10.17161/2tmg0f25OMIM:601068].",
```

Comment:
Should this be formatted like this?
`[@doi:https://doi.org/10.17161/2tmg0f25; @omim:601068]`